woocommerce/woocommerce
# [Experimental] Price Slider block: add colors panel

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.




## ~⚠️  Branched off 52506~

~Please, review https://github.com/woocommerce/woocommerce/pull/52506 first.
I want to make PR shorter when it's possible.~

### Changes proposed in this Pull Request:

This PR adds the `colors` panel (in the block sidebar) to the Price Slider block.





Part of https://github.com/woocommerce/woocommerce/issues/51821
Closes 



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

0. Ensure the experimental blocks are enabled:
    - If you use the WooCommerce Beta Tester plugin, you can enable the experimental blocks in Tools > WCA Test Helper > Features. Toggle the `experimental-blocks` option.
    - If you build this PR locally, build the branch with `pnpm --filter='@woocommerce/plugin-woocommerce' build` command to have experimental blocks available.
1. Ensure to add a few products to your store
2. Ensure to rating products
3. Add the **Product Filters (Experimental)** block to the Product Catalog template.
4. Focus on the Price Slider block.

<img width="353" alt="image" src="https://github.com/user-attachments/assets/e1ef90fc-d3f7-482a-9f64-cf5c06d2b558">

6. Ensure block sidebar is opened/visible
7. Focus the sidebar styles tab
8. Confirm you see the `colors` panel

<img width="288" alt="image" src="https://github.com/user-attachments/assets/8ac48cba-e9e8-4ed1-81e1-5f0a8b0516a9">

9. Set different colors
10. Confirm the slider changes its colors
11. Save the post.
12. Hard refresh
13. Confirm the colors are currently applied in the front- end.

https://github.com/user-attachments/assets/ff02140e-e6d2-4f50-9408-08483ca57baf




### Changelog entry







<details>

<summary>Changelog Entry Details</summary>

#### Significance




#### Type




#### Message 

</details>

<details>

<summary>Changelog Entry Comment</summary>

#### Comment 

</details>
